.github/workflows/format.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           terraform_version: ${{ inputs.terraform_version }}
           terraform_wrapper: false
       - uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          ref: ${{ github.head_ref || github.ref }}
```

Comment:
still fall back to `github.ref` if this isn't a PR event. not sure why anyone would want to run this workflow on a say, push, event, but doesn't hurt to have this fallback I suppose.